libs/community/langchain_community/llms/predibase.py

```diff
@@ -10,10 +10,18 @@ class Predibase(LLM):
 
     To use, you should have the ``predibase`` python package installed,
     and have your Predibase API key.
+
+    The `model` parameter is the Predibase "serverless" base_model ID
+    (see https://docs.predibase.com/user-guide/inference/models for the catalog).
+
+    An optional `adapter_id` parameter is the HuggingFace ID of a fine-tuned LLM
+    adapter, whose base model is the `model` parameter; the fine-tuned adapter
+    must be compatible with its base model; otherwise, an error is raised.
     """
 
     model: str
     predibase_api_key: SecretStr
+    adapter_id: Optional[str]
```

Comment:
```suggestion
    adapter_id: Optional[str] = None
```
for clarity

Reply:
@efriis I incorporated your suggestion regarding the `None` value.